### ISSUE TITLE
Output is completelly blank

### ISSUE THREAD

**Issue by ctsrc** [nidnogg/zeitfetch#14]
On my MacBook Pro 13-inch, M1, 2020 running macOS Ventura 13.4 i installed zeitfetch

```zsh
cargo install zeitfetch
```

```zsh
rustup info
```

```text
Default host: aarch64-apple-darwin
rustup home:  /Users/user/.rustup

installed toolchains
--------------------

stable-aarch64-apple-darwin (default)
1.64-aarch64-apple-darwin

installed targets for active toolchain
--------------------------------------

aarch64-apple-darwin
aarch64-unknown-linux-gnu
wasm32-unknown-unknown

active toolchain
----------------

stable-aarch64-apple-darwin (default)
rustc 1.78.0 (9b00956e5 2024-04-29)
```

```zsh
cargo version
```

```text
cargo 1.78.0 (54d8815d0 2024-03-26)
```

But when I run zeitfetch, the output consists of a bunch of completely blank lines

```zsh
zeitfetch
```

```text



















```

The installed version is the latest one

```zsh
zeitfetch -v
```

```text
zeitfetch v0.1.10
```

**Comment by nidnogg** [comment]
Hi! That's odd. I'll replicate your env here and see if it happens on my mac a well. In the meantime, does the issue persist if you run `sudo zeitfetch` instead?

**Comment by angch** [comment]
I can replicate this, but also that somehow a `cargo install --debug zeitfetch` works. Checking out the code and building both the release and debug version:
```
angch@nardole:~/project/zeitfetch$ ./target/debug/zeitfetch 
                                      angch@nardole 
                         ./+o+-       ------------- 
                 yyyyy- -yyyyyy+      OS: Ubuntu 22.04 
              ://+//////-yyyyyyo      Host: nardole 
          .++ .:/++++++/-.+sss/`      Uptime: 40 minute(s) 
        .:++o:  /++++++++/:--:/-      Kernel: 5.15.146.1-microsoft-standard-WSL2 
       o:+o+:++.`..```.-/oo+++++/     System OS version: 22.04 
      .:+o:+o/.          `+sssoo+/    NB CPUs: 12 
 .++/+:+oo+o:`             /sssooo.   CPU: AMD Ryzen 5 5600H with Radeon Graphics @ 3293 GHz 
 /+++//+:`oo+o               /::--:.  GPU #1: Device 008e 
 \+/+o+++`o++o               ++////.  GPU #2: Device 008e 
  .++.o+++oo+:`             /dddhhh.  Memory: 4842/19677 MiB 
       .+.o+oo:.          `oddhhhh+    
        \+.++o+o``-````.:ohdhhhhh+    ████████████████████████ 
         `:o+++ `ohhhhhhhhyo++os:     ████████████████████████ 
           .o:`.syhhhhhhh/.oo++o`      
               /osyyyyyyo++ooo+++/     
                   ````` +oo+++o\:     
                          `oo++.       
angch@nardole:~/project/zeitfetch$ ./target/release/zeitfetch 



















angch@nardole:~/project/zeitfetch$
```

**Comment by pickfire** [comment]
Same issue on konsole arch. Only release build broke.

**Comment by nidnogg** [comment]
I put a temporary fix there as I'm currently in work hours atm. It should fix the main issue at the cost of ANSI truncating not working 100% for the time being.

**Comment by nidnogg** [comment]
For anyone wanting to try a hand at this while I don't get around to it: I've left a couple of `TO-DOs` in the latest build. Replicating this should be as easy as uncommenting those blocks and running:
```bash
❯ cargo build --release && ./target/release/zeitfetch
```
Then seeing if the output is `size 0 cols 0`. It should read your actual terminal column size.

**Comment by nidnogg** [comment]
Traced down to https://github.com/softprops/termsize/issues/19.

**Comment by angch** [comment]
Practically fixed. Thanks!

**Comment by nidnogg** [comment]
Fixed on latest 1.11.2 release
